modules/services/easyeffects.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.easyeffects;
+
+  presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
+
+in {
+  meta.maintainers = [ maintainers.fufexan ];
+
+  options.services.easyeffects = {
+    enable = mkEnableOption "Easyeffects daemon";
+
+    preset = mkOption {
+      type = types.str;
+      default = "";
+      description = ''
+        Which preset to use when starting easyeffects.
+        Will likely need to launch easyeffects to initially create preset.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    # running easyeffects will just attach itself to gapplication service
+    # at-spi2-core is to minimize journalctl noise of:
+    # "AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files"
+    home.packages = with pkgs; [ easyeffects at-spi2-core ];
+
+    # Will need to add `services.dbus.packages = with pkgs; [ gnome.dconf ];`
+    # to /etc/nixos/configuration.nix for daemon to work correctly
```

Comment:
This feels like documentation that should be in the option, not in the source. I think we should add it to the `enable` option. I know that `pulseeffects` doesn't do this, but let's at least do it correctly for this one.